packages/react-test-renderer/npm/shallow.js

```diff
@@ -1,3 +1,9 @@
 'use strict';
 
-module.exports = require('react-shallow-renderer');
+function ReactShallowRenderer() {
+  throw new Error(
+    'react-test-renderer is deprecated. To use react-test-renderer/shallow, import react-shallow-renderer directly. See https://react.dev/warnings/react-test-renderer'
```

Comment:
Nit: "deprecated" usually doesn't mean "removed" but in this case, this is removed. I would say something like this instead:

> react-test-renderer/shallow has been removed. See https://react.dev/warnings/react-test-renderer.

I'm not sure we should direct to react-shallow-renderer in the error message, or if we should mention it on the warning page instead. Really, people shouldn't be using shallow rendering at all, and direct people to that package would lead them in the wrong direction.

Reply:
I can move more detail to the warning page. My thinking for letting people know about the package movement is just to minimize the amount of blockers for adopting 19. If an app happens to use a bunch of shallow renderer in their test suite, hopefully they can drop in the original package we were reexporting and work on an incremental migration instead of being blocked completely.